BlazarService/src/main/java/com/hubspot/blazar/resources/BranchResource.java

```diff
@@ -71,6 +78,23 @@ public BranchResource(BranchService branchService,
     return branchService.get(branchId);
   }
 
+  @Path("/{branchId}/status")
+  public Optional<BranchStatus> getBranchStatusById(@PathParam("branchId") int branchId) {
+    return branchStatusService.getBranchStatusById(branchId);
+  }
+
+  @GET
+  @Path("/state/{branchId}/modules")
+  @PropertyFiltering
+  public Set<ModuleState> getAllModuleStatesForBranch(@PathParam("branchId") int branchId) {
```

Comment:
@gchomatas I re-wrote this to use the new `BranchStatusService` so that we could get rid of the old code that just produced module states, and still return the old-style data.